sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/implementation/models/AttestationSignerCollectionImpl.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.attestation.implementation.models;
+
+import com.azure.core.annotation.Immutable;
+import com.azure.security.attestation.models.AttestationSigner;
+import com.azure.security.attestation.models.AttestationSignerCollection;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * An AttestationSignerCollection represents a collection of {@link AttestationSigner} objects.
+ */
+@Immutable
+public class AttestationSignerCollectionImpl implements AttestationSignerCollection {
+    private List<AttestationSigner> signers;
+
+    public AttestationSignerCollectionImpl(List<AttestationSigner> newSigners) {
+        signers = new ArrayList<>(newSigners);
```

Comment:
Since this is an output model, we don't have to worry about `newSigners` being mutated externally. We can do the following instead.
```suggestion
        this.signers = newSigners;
```

Reply:
For some reason, I thought I was going to hit a checkstyle violation because of this. But apparently not.